### PR TITLE
Add ml-hyperopt to example pipelines

### PR DESCRIPTION
**Pull request by bentsherman**

### CODE REVIEW

File: content/example5.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* Java 11 (or higher)
+* Anaconda
+
+Install Nextflow by entering the following command in the terminal:
+
+    $ curl -fsSL get.nextflow.io | bash
+
+Then launch the pipeline with this command:
+
+    $ nextflow run ml-hyperopt -profile conda
```
Comment (pditommaso) [inline content/example5.md:69]:
```suggestion
    $ nextflow run ml-hyperopt -profile wave
```

Wave first!

Comment (bentsherman) [reply content/example5.md:69]: Lol okay, then I need to specify that you need a Tower Cloud account

Comment (pditommaso) [reply content/example5.md:69]: not really. That's needed only for high rate limits. Wave works for anonymous users

Comment (bentsherman) [reply content/example5.md]: Cool, I did not know that

**Comment by pditommaso** [comment]
Great, thanks

**Pull request merged**